posts/2023/organizing-farm-data/index.md

```diff
@@ -0,0 +1,117 @@
+> 
+> Next, take that statement and imagine what data it implies. For instance the grazing rotation example might require knowing the timing of which (and how many) animals
+> were in which pastures along with - at least - starting and ending weights for those animals. The plant starts example might require knowing the dates when new batches
+> of starts were planted along with the quantity and sales records for the resulting starts.
+> 
+> It might also be important to consider the ways in which the data will need to be accessed to achieve the goal. Maybe the data gets used once a month as some sort of
+> report or maybe it is accessed on a day-to-day basis. This can help inform the granularity of the data and the selection of data formats/tools.
+> 
+> ## Data Structure and Re-usability
+> 
+> This next part is arguably the hardest because it requires researching (and possibly experimenting with) the available tools and data model choices. It also involves
+> weighing the trade-offs between coarser or finer grained data. Here we are discussing use-cases where farmOS is a good fit, but it is important to keep in mind that
+> for other use-cases accounting, GIS, or even spreadsheet/paper tools might be a better fit.
+> 
+> The [farmOS data model](https://farmos.org/model/) provides for representing your growing things and related stuff using assets. It also provides logs for recording
+> events. The basic ideas of how to use those primitives (and a few others) are described in the [farmOS user guide](https://farmos.org/guide/). Armed with that information,
+> we can start to decide what structure and granularity will help meet your goals.
+> 
+> In the grazing rotation example we need to decide whether to represent the herd/flock/etc as individual animal assets or as a single group asset. Correspondingly
+> the weights might be recorded as an animal count, total weight, and average weight for the whole group or as individual weight logs for each animal. One thing that
+> can help with that decision is determining whether there are other ways to re-use the data. e.g. Would having individual animal pasture locations and weight history
+> be helpful in finding animals or making care decisions day-to-day?
+> 
+> farmOS is very unopinionated about how one uses its data model. This can be helpful but also means we need to exercise some creativity when choosing the best fit for
+> any given use-case. In the employee tool and material finding example above, it might be tempting to track the locations of each tool or material when employees move them
+> around, but it might not be realistic to expect employees to be updating those locations very well in real-time. Instead, you could choose to just model the normal storage
+> location for the tool or material assets that way employees know where to look first and where to put the materials back when they're done with them. By also referencing
```

Comment:
This reads a bit awkward. Maybe just missing punctuation between "material assets" and "that way"?

Reply:
Good catch!